### ISSUE TITLE
Layout tab : custom command not saved

### ISSUE THREAD

**Issue by LionelHoudelier** [gnome-terminator/docs#21]
In Layout tab:
- add a new layout from existing window with different terminals
- or edit one
- add a custom command for a terminal
- click save
- then the command disappears 
any idea why?

**Comment by mattrose** [comment]
I did look at this a while back, and it has to do with the layout save procedure.  There is a github issue about it, but I'm mostly concentrating on trying to update to GTK4 right now.

**Comment by LionelHoudelier** [comment]
yes. because we cannot change the saved layout, so the save action cancels the changed options (profile, custom command, working folder). sad. It is a wonderful app in development! Thank you so much.
I am looking at the code to see where to change the action.
HALF-SOLVED:
Do not press "save".

**Comment by apnea** [comment]
The issue is actually that the button should not say `Save` but something like  `Refresh`  or  `Reset`  :

- the behaviour of this dialog is that changes get saved when the user hits enter, by invoking `self.config.save()`
- while the button click in question resets any changes

Therefore a  `Save`  button is not needed and misleading. The existing code and glade file uses the term `layoutrefresh` to refer to the button behaviour, so this looks like a simple oversight in the button property in preferences.glade.

I'll do a pull request with the change from `gtk-save` to `gtk-refresh` to bring the glade file in line with the code.
